features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/metric/ViewInitializationMetricsState.kt

```diff
@@ -0,0 +1,55 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+package com.datadog.android.rum.internal.metric
+
+internal data class ViewInitializationMetricsState(
+    val initializationTime: Long?,
+    val config: ViewInitializationMetricsConfig,
+    val noValueReason: NoValueReason?
+)
```

Comment:
> [!NOTE]
Nitpick, feel free to disregard. Although both the metrics inv

Reply:
Sorry, I didn't get the question, what do you mean here?